Lib/functools.py

```diff
@@ -402,7 +398,7 @@ def _method(cls_or_self, /, *args, **keywords):
     def __get__(self, obj, cls=None):
         get = getattr(self.func, "__get__", None)
         result = None
-        if get is not None and not isinstance(self.func, partial):
+        if get is not None:
```

Comment:
It was needed to avoid emitting a warning for `partialmethod(partial(...))`, even if the final behavior is not changed.